src/documentation/.gitignore

```diff
@@ -88,3 +88,6 @@ sw.*
 
 # Vim swap files
 *.swp
+
+# Local sensative data
+.npmrc
```

Comment:
Great idea! I think we should consider including that in other .gitignore files 👏